cudax/include/cuda/experimental/data_manipulation.cuh

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __CUDAX_DATA_MANIPULATION__
```

Comment:
Should we rather name that `algorithm`?

Reply:
It possible, but it seemed a bit ambitious to call these two `algorithm`. We should also think how is this going to play out long term along with single-threaded copy/fill from libcu++. But I can change it

Reply:
more forward-looking than ambitious. i like `algorithm` for this.